remote_attestation/proto/remote_attestation.proto

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.remote_attestation;
+
+option java_multiple_files = true;
+option java_package = "oak.remote_attestation";
+
+/// Placeholder implementation of TEE report for remote attestation.
+/// https://www.amd.com/system/files/TechDocs/56860.pdf#page=39
+///
+/// TODO(#1867): Add remote attestation support and use real TEE reports.
```

Comment:
Note that in practice a claim / assertion is probably just going to be an opaque byte blob as returned by the workload attestation, and it will have various layers of attestations. Anyways, we will see when we get there.